packages/select/src/components/multi-select/multiSelect2.tsx

```diff
@@ -241,6 +241,7 @@ export class MultiSelect2<T> extends AbstractPureComponent2<MultiSelect2Props<T>
             };
             return (
                 <div
+                    aria-autocomplete="list"
```

Comment:
this looks like a good addition... I think we should also set this in other places we use `role="combox"`... Select2 and Suggest2

Reply:
We already have that on `Suggest2`. On `Select2`, it didn't seem appropriate on the combobox since you can't type into that. I put it on the filter input though.